src/components/BottomNav.js

```diff
@@ -1,23 +1,54 @@
 import React from 'react';
-import { NavLink } from 'react-router-dom';
 
-function BottomNav() {
+import { Link } from 'react-router-dom';
+import { makeStyles } from '@material-ui/core/styles';
+import AppBar from '@material-ui/core/AppBar';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+import FormatListBulletedIcon from '@material-ui/icons/FormatListBulleted';
+import AddCircleOutlineIcon from '@material-ui/icons/AddCircleOutline';
+
+const useStyles = makeStyles(theme => ({
+  root: {
+    minWidth: 375,
+    maxWidth: 3000,
+    height: 30,
+  },
+}));
+
+export default function FullWidthTabs() {
+  const classes = useStyles();
+  const [value, setValue] = React.useState(0);
+
+  const handleChange = (event, newValue) => {
+    setValue(newValue);
+  };
+
   return (
-    <nav>
-      <ul>
-        <li>
-          <NavLink exact to="/list" activeClassName="selected-view">
-            List
-          </NavLink>
-        </li>
-        <li>
-          <NavLink exact to="/add-item" activeClassName="selected-view">
-            Add Item
-          </NavLink>
-        </li>
-      </ul>
-    </nav>
+    <div className={classes.root}>
+      <AppBar position="static" color="default">
+        <Tabs
+          value={value}
+          onChange={handleChange}
+          indicatorColor="primary"
+          textColor="primary"
+          variant="fullWidth"
+          aria-label="full width tabs example"
+        >
+          <Tab
+            icon={<FormatListBulletedIcon />}
+            aria-label="list"
```

Comment:
Nice. Maybe shopping list as the aria-label? Imagine if you were navigating the app without using your eyes...what information would you need to understand what this item does?